azure-devops/run_build.yml

```diff
@@ -0,0 +1,51 @@
+parameters:
+  targetPlatform: 'x64'
+
+jobs:
+- job: ${{ parameters.targetPlatform }}
+  pool:
+    name: STL
+
+  variables:
+    vcpkgLocation: '$(Build.SourcesDirectory)/vcpkg'
+    vcpkgResponseFile: $(Build.SourcesDirectory)/vcpkg_windows.txt
+  steps:
+    - checkout: self
+      submodules: recursive
+    - task: BatchScript@1
+      displayName: 'Enforce clang-format'
+      condition: eq('${{ parameters.targetPlatform }}', 'x86')
+      inputs:
+        filename: 'azure-devops/enforce-clang-format.cmd'
+        failOnStandardError: true
+      # Install VS 2019 preview on Microsoft Hosted Agents only.
+    - task: PowerShell@2
+      displayName: 'Install MSVC preview'
+      condition: or(contains(variables['Agent.Name'], 'Azure Pipelines'), contains(variables['Agent.Name'], 'Hosted Agent'))
+      inputs:
+        targetType: filePath
+        filePath: $(Build.SourcesDirectory)/azure-devops/install_msvc_preview.ps1
+    - task: CacheBeta@0
+      displayName: Cache vcpkg
+      inputs:
+        key: $(vcpkgResponseFile) | $(Build.SourcesDirectory)/.git/modules/vcpkg/HEAD
```

Comment:
Verifying my understanding: Does this mean the cache is keyed off the concatenation of these two filenames, or the concatenation of the contents of the two named files?

Reply:
Hash of the files is computed and used as a key. Cache is immutable once created with given key. You can add further segment separated with pipe.

Reply:
Thanks!